package.json

```diff
@@ -83,7 +83,7 @@
     "rollup-plugin-vue": "3.0.0",
     "rollup-watch": "4.3.1",
     "vue": "2.5.17",
-    "vue-autosuggest": "^1.5.0",
+    "vue-autosuggest": "1.5.0",
```

Comment:
Btw 1.6 is out now, maybe not relevant here https://github.com/Educents/vue-autosuggest/releases/tag/v1.6.0